### PR TITLE
Add net.krafting.Reddy

### PR DESCRIPTION
**Pull request by Krafting**
## What is Reddy

Reddy is a tool to manually repost images from Reddit posts to Lemmy

### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. 
- [x] The domain used for the application ID is controlled by the application developers either directly 
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by Krafting** [comment]
bot, build net.krafting.Reddy

**Comment by Krafting** [comment]
bot, build net.krafting.Reddy

**Comment by Krafting** [comment]
bot, build net.krafting.Reddy

**Comment by Krafting** [comment]
bot, build net.krafting.Reddy

**Comment by hfiguiere** [comment]
> I have [built](https://docs.flathub.org/docs/for-app-authors/submission/#before-submission) and tested the submission locally.

Obviously you didn't. Yet you answered yes.

File: net.krafting.Reddy.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  - name: Reddy
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 Reddy.py /app/bin/Reddy.py
+      - install -Dm755 functions/download.py /app/bin/functions/download.py
+      - install -Dm755 functions/fetch_post.py /app/bin/functions/fetch_post.py
+      - install -Dm755 functions/post_post.py /app/bin/functions/post_post.py
+      - install -Dm755 functions/utils.py /app/bin/functions/utils.py
+      - install -Dm 0644 images/net.krafting.Reddy.svg /app/share/icons/hicolor/256x256/apps/net.krafting.Reddy.svg
```
Comment (hfiguiere) [inline net.krafting.Reddy.yml:28]: svg is NOT 256x256

**Comment by bbhtt** [comment]
bot, build net.krafting.Reddy

**Pull request closed without merging**